modules/custom/az_paragraphs/az_paragraphs.install

```diff
@@ -79,3 +79,66 @@ function az_paragraphs_update_920301(&$sandbox) {
   return t('Breakpoint settings updated on %count paragraphs.', ['%count' => $sandbox['updated_count']]);
 
 }
+
+/**
+ * Replace broken migrated paragraph_behavior settings with defaults.
+ */
+function az_paragraphs_update_920601(&$sandbox) {
+
+  if (!isset($sandbox['total'])) {
+    $count = \Drupal::entityQuery('paragraph')
+      ->accessCheck(FALSE)
+      ->count()
+      ->execute();
+    // Set the total number of paragraphs to be updated.
+    $sandbox['total'] = $count;
+    // Set current to 0.
+    $sandbox['current'] = 0;
+    // Set updated_count to 0.
+    $sandbox['updated_count'] = 0;
+    // If there are no paragraphs to be updated, set the finished flag to 1.
+    if (empty($sandbox['total'])) {
+      \Drupal::messenger()
+        ->addMessage('No broken behavior settings found.');
+      $sandbox['#finished'] = 1;
+      return;
+    }
+  }
+  // Set the number of paragraphs to be updated per batch.
+  $paragraphs_per_batch = 25;
+  // Get the paragraph ids to be updated.
+  $paragraph_ids = \Drupal::entityQuery('paragraph')
+    ->accessCheck(FALSE)
+    // Update 25 paragraphs per batch.
+    ->range($sandbox['current'], $sandbox['current'] + $paragraphs_per_batch)
+    ->execute();
+
+  $paragraphs = \Drupal::entityTypeManager()->getStorage('paragraph')->loadMultiple($paragraph_ids);
```

Comment:
Loads 25 at a time, instead of 1 at a time in the foreach, not sure the best approach.

Reply:
> Loads 25 at a time, instead of 1 at a time in the foreach, not sure the best approach.

I like the way you have it.  I think limiting the batch to 25 (or possibly even less?) at a time is a good idea to prevent timeouts.